app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/componentHealthCheckControls/componentHealthCheckControls.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { FieldArray } from 'redux-form';
+import { validate } from 'common/utils';
+import { COMPONENT_HEALTH_CHECK } from 'common/constants/widgetTypes';
+import { FieldProvider } from 'components/fields/fieldProvider';
+import { FiltersControl, InputControl } from '../controls';
+import { ITEMS_INPUT_WIDTH } from '../constants';
+import { AttributesFieldArrayControl } from '../controls/attributesFieldArrayControl/attributesFieldArrayControl';
+import styles from '../widgetControls.scss';
+
+const cx = classNames.bind(styles);
+
+const MAX_ATTRIBUTES_AMOUNT = 10;
+const DEFAULT_PASSING_RATE = '100';
```

Comment:
Why is it called not `DEFAULT_ITEMS_COUNT`? According to its usage as `itemsCount`